config/locales/cy/cy.yml

```diff
@@ -974,6 +974,7 @@ cy:
             no_acas_number_reason:
               blank: "Dewiswch reswm o'r rhestr."
               inclusion: ""
+            invalid_cdd_acas_exemption_reason: 'activemodel.errors.models.respondent.attributes.no_acas_number_reason.blank'
```

Comment:
Have you looked at the error message that is output from this ?  I think you need a colon in front of the 'activemodel.....' - the colon says that it is a reference to another entry rather than the value being 'activemodel.errors.models.respondent.attributes.no_acas_number_reason.blank'